DEVELOPMENT.md

```diff
@@ -0,0 +1,69 @@
+# Developing MXNet
+
+## Build MXNet from source with CMake
+
+It's recommended that you install CMake and Ninja.
+
+Chose the options that you want to compile with in a similar fashion as above. In particular this one is a debug CPU
+build in OSx.
+
+```
+mkdir build && cd build
+cmake -DUSE_CUDA=OFF -DUSE_OPENCV=OFF -DUSE_OPENMP=OFF -DCMAKE_BUILD_TYPE=Debug -GNinja ..
```

Comment:
Why are you turning off OpenCV in this example when all the other build examples have it turned on? Maybe explain what you're doing here...